run_aks_test.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
```

Comment:
Is this file required to be merged into master?

Reply:
it is example for aks testing